_grab_ui.py

```diff
@@ -826,7 +689,7 @@ def ospec():
         lines = (sys.stdin.buffer.read().decode('utf-8')
                  .split('\n')[:-1])  # last line ends with \n, too
         sys.stdin = tty
-        opts = parse_opts()
+        opts = load_config()
```

Comment:
The `load_config` function accepts a list of `paths` to parse and load, but you pass an empty list here, and I don’t see any magic [in kitty.config.utils.load_config][base] that would find our config. Are we just running with defaults?

[base]: https://github.com/kovidgoyal/kitty/blob/v0.21.2/kitty/conf/utils.py#L218

Reply:
Seems indeed to be the case, do you know that `paths` should be? The paths to places we look for user configs?

Reply:
In 00d19bd1aa236db49550c6de940df599b90d8dd5 I now changed it to pass in the path to the user config and it does seem to work :)

Reply:
I call it the shoelace tying principle: Before you can proceed with a refactoring, you usually have to untie or remove some things, but afterwards you [go through things sticking out or lying on your desk][paths] and make sure that each one is reattached to its new appropriate place.

[paths]: https://github.com/yurikhan/kitty_grab/pull/14/files#diff-8087c975402900536bc73314799fb77e1dd7f11e5bc24ddda2f69e8f571a9db3L531-L532

Rationale for these paths: the first is a system-wide [standard][xdg] location, suitable for a machine admin to configure for all users; and the second is a standard user-specific one. [Kitty itself looks for its configs there][config].

[xdg]: https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
[config]: https://sw.kovidgoyal.net/kitty/invocation.html#cmdoption-kitty-config

Reply:
That's a good principle. I guess in this case I wasn't really sure how to untie the laces in the first place so I cut them loose with a knife and tried to tie together the ends the best I could haha :)

Anyway, updated in a2c8a5907fef03951fd26039f05ba5dc2625b880